www/index.php

```diff
@@ -94,9 +99,9 @@ function default_status_time() {
  */
 foreach ($config['feature'] as $feature) {
     if ($feature['enabled'] == "on") {
-        error_log("Including Feature {$feature['name']}");
-        include $feature['name'] . '/lib.php';
-        include $feature['name'] . '/routes.php';
+        $log->debug("Including Feature {$feature['name']}");
```

Comment:
It seems the log levels help here.  We can set the log level when/right after initializing $app.  Something like ```$log->setLevel(\Slim\Log::DEBUG);```  I can add this as well